spec/features/admin/orders_spec.rb

```diff
@@ -30,6 +30,19 @@ def new_order_with_distribution(distributor, order_cycle)
     click_button 'Next'
   end
 
+  scenario "order cycles appear in descending order by close date" do
+    create(:simple_order_cycle, name: 'Two', orders_close_at: 2.weeks.from_now)
+    create(:simple_order_cycle, name: 'Four', orders_close_at: 4.weeks.from_now)
+    create(:simple_order_cycle, name: 'Three', orders_close_at: 3.weeks.from_now)
+
+    login_to_admin_section
+    visit 'admin/orders'
+
+    open_select2('#s2id_q_order_cycle_id_in')
+
+    find('#q_order_cycle_id_in', visible: :all)[:innerHTML].should =~ /.*One.*Two.*Three.*Four/m
```

Comment:
Can we please use the `expect` RSpec syntax? Although there's no style guide for it yet, we use this one now.

Reply:
Ok sounds good. Amended.